.github/workflows/check-change.yml

```diff
@@ -0,0 +1,30 @@
+name: Check change
+on:
+  pull_request:
+
+jobs:
+  check-change:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '14'
+
+      - uses: actions/github-script@v6
+        with:
+          script: |
+            const beachballVersion = require('./package.json').devDependencies.beachball;
+            if (typeof beachballVersion !== 'string') {
+              core.setFailed("Please check 'package.json', it should include 'beachball' in 'devDependencies'")
+            } else {
+              console.log(`Using beachball@${beachballVersion}...`)
+              await exec.exec('yarn', ['global', 'add', `beachball@${beachballVersion}`]);
+            }
```

Comment:
This could be moved to a separate file, but keeping it inside the workflow makes it self descriptive.